src/pspm_compute_visual_angle.m

```diff
@@ -93,92 +89,75 @@
     if strcmpi(options.eyes, 'all') || strcmpi(options.eyes(1), eye)

```

Comment:
If I am not mistaken, `infos.source.eyesObserved` would contain one of `l`, `r` or `lr`. So the `strcmpi` check against `left`, `right` or `all` won't work. Could you fix this naming discrepancy?

Reply:
It will work I think, no ? Since `options.eyes` is an option specific to this function which can be 'right', 'left' or 'all', i.e. `options.eyes(1)` would then be respectively 'r', 'l' or 'a'. So the first two would work with `infos.source.eyesObserved`.
But I agree that this function do not follow the convention of `infos.source.eyesObserved`. Shall I change the convention for `options.eyes` ? (i.e. make it 'r', 'l' or 'lr')

Reply:
Ah now I see. I think having the same naming convention everywhere is a good idea, so yes please change it to 'l', 'r', 'lr'.

Reply:
Copy that